modules/acquia_cms_tour/css/acquia_cms_installation_wizard.css

```diff
@@ -0,0 +1,164 @@
+.acms-installation-wizard {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+.acms-installation-wizard .ui-dialog-titlebar .ui-dialog-titlebar-close {
+  top: 24%;
+}
+.acms-installation-wizard .ui-dialog-titlebar .ui-dialog-title {
+  display: inline-block;
+  margin-top: 61px;
+  font-size: 32px;
+  line-height: 1.3;
+  color: #232429;
+}
+.acms-installation-wizard .ui-widget-header.ui-dialog-titlebar {
+  height: 125px;
+  background-color: #fff;
+  padding-left: 48px;
+}
+.acms-installation-wizard .messages-list {
+  margin: 0;
+}
+.acms-installation-wizard #drupal-modal {
+  padding-left: 0;
+  padding-bottom: 0;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard {
+  display: flex;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .form-checkbox {
+  background-color: #003cc5;
+  border-color: #003cc5;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar {
+  flex: 1 1 34%;
+  font-size: 16px;
+  line-height: 1.5;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar ol {
+  margin-left: 0;
+  padding-left: 0;
+  list-style-type: none;
+  counter-reset: qa;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar .item {
+  padding: 16px 0;
+  align-items: center;
+  display: flex;
+  margin-left: 0;
+  padding-left: 48px;
+  margin-bottom: 0;
+  counter-increment: qa;
+  color: #232429;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar li:before {
+  content: counter(qa);
+  padding-right: 10px;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar .step-complete {
+  color: #919297;
+}
+.acms-installation-wizard .acquia-cms-tour-installation-wizard .tour-sidebar .step-complete::after {
+  content: "";
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  background: url(/sites/all/modules/acquia_cms_tour/assets/images/check.svg);
```

Comment:
@chandan-singh7929 I think, we should avoid adding absolute path as this module can be placed at some other location as well.